docs/plugins/auto-trim.md

```diff
@@ -0,0 +1,54 @@
+# Auto Trim
+
+By default, Vento doesn't automatically try to trim the whitespace from around tags. Using the
+`autoTrim` plugin, you can enable this behavior.
+
+## Usage
+
+`autoTrim` comes included with Vento, so you don't need to install any additional packages.
+Simply import it and use it in your instance.
+
+```js
+import autoTrim from "https://deno.land/x/vento@v0.10.3/plugins/autoTrim.ts";
+// import autoTrim from "ventojs/plugins/autoTrim.js";
```

Comment:
to avoid showing urls including the version (that are going to be outdated), we can simply do `vento/plugins/auto_trim.ts`.